docs/accessControl/index.md

```diff
@@ -350,3 +354,20 @@ The student's access will expire if they exceed the `timeLimitMin` minute durati
 ```
 
 This `allowAccess` directive is suitable for a `Homework` assessment. Students can see that the homework exists before Oct 12th, but because it is `"active": false` they won't be able to see the questions or start the homework. They can start the homework anytime from Oct 12th onwards, and the homework goes through four different stages: (1) they will earn a bonus 10% if they complete the homework with full points before Oct 15th, (2) they get full credit until the due date of Oct 18th, (3) they can complete the homework up to a week late (Oct 25th) for 80% credit, and (4) they will be able to see the homework and submit answers for feedback, but not earn more points, at any time after that. The homework visibility will also be limited by the [course instance access rules](#course-instance-example), so the homework won't be accessible in any way outside the course instance date range.
+
+## Preview access control
+
+Click on your user in the top right corner to see the role picker dropdown. This will allow you to preview the access control for different roles and users.
+
+![Showing the role picker dropdown](role_picker.png){ width=50% }
```

Comment:
I don't like specifying explicit widths as things look weird on narrow viewports:

<img width="470" alt="Screenshot 2025-04-17 at 11 29 15" src="https://github.com/user-attachments/assets/6d96967c-984c-4d7e-aba2-bea11af95b71" />

Is `max-width` supported? I usually reach for that for responsive images.

Reply:
Changed in a9896720b43906e1f1f7625571275bcf5b08082a.

Reply:
Worth mentioning that I made the same mistake on pretty much every other image on the docs.

Reply:
Opened an issue in #11802